docs-v2/pages/connect/components.mdx

```diff
@@ -0,0 +1,501 @@
+import { Steps, Tabs } from 'nextra/components'
+import ArcadeEmbed from '@/components/ArcadeEmbed'
+import Callout from '@/components/Callout'
+import Image from 'next/image'
+
+# Running Components for Your End Users
```

Comment:
```suggestion
# Running components for your end users
```

(see https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings)